tiny_dnn/tiny_dnn.h

```diff
@@ -102,6 +103,8 @@ using fc = tiny_dnn::fully_connected_layer;
 
 using dense = tiny_dnn::fully_connected_layer;
 
+using zreo_pad = tiny_dnn::zero_pad_layer;
```

Comment:
I've found  a typo. `zreo_pad` should be `zero_pad`.

Reply:
OMG.
Thank you!